eng/Version.Details.xml

```diff
@@ -37,7 +29,7 @@
       Win-x64 is used here because we have picked an arbitrary runtime identifier to flow the version of the latest NETCore.App runtime.
       All Runtime.$rid packages should have the same version.
     -->
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="6.0.2" CoherentParentDependency="Microsoft.AspNetCore.Testing">
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="6.0.2">
```

Comment:
Is there a runtime -> razor-tooling subscription now?

Reply:
I'm not seeing one. Suggest removing this entry and putting the version property in a "manual" section of eng/Versions.props.

Reply:
There is now:

```
https://github.com/dotnet/runtime (.NET 7.0.1xx SDK) ==> 'https://github.com/dotnet/razor-tooling' ('main')
  - Id: f572d8e1-e480-4960-2c46-08d98f562a28
  - Update Frequency: EveryBuild
  - Enabled: True
  - Batchable: True
  - PR Failure Notification tags:
  - Merge Policies:
    Standard
  - Last Build: 20220211.3 (d5d2e4da9e1371533c27d073a09012ff96aeeffb)
```

Originally it was `aspnetcore`. Updated to `sdk` earlier in this thread:

> Updated SourceDepositoryUrl of the f572d8e1-e480-4960-2c46-08d98f562a28 subscription to use the sdk repo instead of aspnetcore.

With this update its runtime -> tooling.

Reply:
Actually, why handle the runtime separately at all❔ Just bump the SDK entries in global.json to 6.0.102 and try to remember to do something similar about monthly.

Reply:
(We should bump the SDK in razor-compiler too)

Reply:
Updated to use release/6.0 channel `(3800) https://github.com/dotnet/runtime @ release/6.0 -> .NET 6`

```
https://github.com/dotnet/runtime (.NET 6) ==> 'https://github.com/dotnet/razor-tooling' ('main')
  - Id: f572d8e1-e480-4960-2c46-08d98f562a28
  - Update Frequency: EveryBuild
  - Enabled: True
  - Batchable: True
  - PR Failure Notification tags:
  - Merge Policies:
    Standard
  - Last Build: 20220211.3 (d5d2e4da9e1371533c27d073a09012ff96aeeffb)
```

Reply:
Again this is silly given the SDK updates every time the runtime you care about does. Remove this entry entirely, along w/ the version in eng/Versions.props. All that's needed is the SDK version in global.json.

Reply:
```
darc delete-subscriptions --id  f572d8e1-e480-4960-2c46-08d98f562a28
Will delete the following 1 subscriptions...
  https://github.com/dotnet/runtime (.NET 6) ==> 'https://github.com/dotnet/razor-tooling' ('main')
Continue? (y/n) y
Deleting 1 subscriptions...done
```

Going to leave the rest as is for now. Don't want to block the VS4Mac insertion on these additional changes.

Reply:
Alright. Bit wasteful to validate another PR but could you put that up soon❔ Confusing to have this entry but no subscription to match.

Reply:
Just added it here given we're blocked on the test changes: https://github.com/dotnet/razor-tooling/pull/6088/commits/0be1f4cd671ca99202c40294e90fa530867ed9f5